lib/ensureStringLength.js

```diff
@@ -0,0 +1,23 @@
+const DEFAULT_ERROR = 'STRING_LENGTH_NOT_IN_SPECIFIED_RANGE'
+const DEFAULT_MIN = 1
+const DEFAULT_MAX = Number.MAX_SAFE_INTEGER
+
+module.exports = (unsafeString, options = {}) => {
+  if (unsafeString === undefined || unsafeString === null || typeof unsafeString !== 'string') {
+    // bail if it's not a string or no value at all
+    return
+  }
+
+  const { length } = unsafeString
+  const {
+    max = DEFAULT_MAX,
+    min = DEFAULT_MIN,
+    error = DEFAULT_ERROR
+  } = options
+
+  if (Number.isInteger(max) && length > max) {
+    throw new Error(error)
+  } else if (Number.isInteger(min) && length < min) {
+    throw new Error(error)
+  }
```

Comment:
why not `parseInt(max)`?

Reply:
parseInt is doing too much magic for my taste.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt (see examples).

Reply:
Thats the real beauty of javascript. Meh. I just think it's more robust to parse an int somehow than to just ignore the limits if they are not-exactly-an-int. What about the `filterInt` function from the link you posted?

Reply:
Aight, I take that one